app/templates/views/pricing/letter-pricing.html

```diff
@@ -0,0 +1,57 @@
+{% extends "content_template.html" %}
+
+{% from "components/content-metadata.html" import content_metadata %}
+{% from "components/table.html" import mapping_table, row, text_field, row_heading %}
+
+{# Used by the content_template.html layout, prefixes the "navigation" accessible name #}
+{% set navigation_label_prefix = 'Pricing information' %}
+
+{% block per_page_title %}
+  Letter pricing
+{% endblock %}
+
+{% block content_column_content %}
+
+  <h1 class="heading-large">Letter pricing</h1>
+
+    {{ content_metadata(
+      data={
+        "Last updated": "24 January 2023"
+      }
+    ) }}
+
+  <p class="govuk-body">The cost of sending a letter depends on the postage you choose and how many sheets of paper you need.</p>
+
+  <p class="govuk-body">Prices include:</p>
+  <ul class="govuk-list govuk-list--bullet">
+     <li>paper</li>
+     <li>postage</li>
+     <li>double-sided colour printing</li>
+     <li>C5 size envelopes with an address window</li>
+  </ul>
+
```

Comment:
I borrowed this line from the `Upload a letter` page, because ought to use it somewhere prominent - and this seems like a sensible place.

```suggestion

<p class="govuk-body">Letters must be 10 pages or less (5 double-sided sheets of paper).</p>

```